doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -350,6 +350,7 @@ Serial LTE modem
 
   * Support for the :file:`overlay-native_tls.conf` configuration file with the ``thingy91/nrf9160/ns`` board target.
   * Support for deprecated RAI socket options ``AT_SO_RAI_LAST``, ``AT_SO_RAI_NO_DATA``, ``AT_SO_RAI_ONE_RESP``, ``AT_SO_RAI_ONGOING``, and ``AT_SO_RAI_WAIT_MORE``.
+  * The ``#XCARRIERCFG="bootstrap_smartcard"`` AT command.
```

Comment:
We would also require an entry for LwM2M carrier library. For example : https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.7.0.html#binary_libraries.

Reply:
Discussed offline and will be covered in the release notes PR.